image/cli/mascli/templates/pipelinerun.yaml

```diff
@@ -75,6 +75,26 @@ spec:
 
     # Dependencies - MongoDb
     # -------------------------------------------------------------------------
+
```

Comment:
See that parameters are under "MongoDB" section, could you please move it above wher "Dependencies - Db2" are? Then we can add them as "Dependencies - JDBC Settings" as used in the pipeline parameters

Reply:
done